addons/edi/models/edi_issues.py

```diff
@@ -35,6 +35,127 @@ class ProjectTask(models.Model):
     )
 
 
+class BaseIssue(models.AbstractModel):
+    """Base Issue-Tracked Object"""
+
+    # To consider: Would be good to move this abstract class in a module that only depends
+    # on project module in the future. So if we want to replicate this solution not to depend on
+    # EDI module by force.
+    _name = "base.issues"
+    _inherit = "mail.thread"
+    _description = "Base Issue Object"
+
+    # Empty list variable, will be overriden in the specific models where used.
+    ISSUES_FIELD_MAP = []
+
+    issue_ids = fields.One2many(
```

Comment:
Maybe a comment to add the inverse name wherever issue_ids will be used in models.Model classes

Reply:
There is no need, thanks.